specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/examples/DeletedFileSharesList.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res9290",
+    "accountName": "sto1590",
+    "$expand": "deleted",
+    "api-version": "2019-06-01",
+    "monitor": "true"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
```

Comment:
For the response of list operation,  should we return an array directly not a json with array in "value"?

Reply:
It is by design. We may have nextLink alongside with value. We return limited number of shares, depends on the page size.